src/design-guidelines/action-buttons/examples/labels-explicit.tsx

```
@@ -8,38 +8,32 @@ import { Component, h } from '@stencil/core';
-                    <limel-header
-                        icon="brake_warning"
-                        heading="Don't"
-                    ></limel-header>
-                    <div class="fake-dialog-container shows-full-dialog">
-                        <div class="fake-dialog">
-                            <p>Discard this draft?</p>
-                            <div class="action-bar">
-                                <limel-button label="No" primary={true} />
-                                <limel-button label="Yes" />
-                            </div>
+            <limel-example-do-do-not>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)